pkg/reconciler/buildrun/resources/results_test.go

```diff
@@ -123,12 +123,22 @@ var _ = Describe("TaskRun results to BuildRun", func() {
 						Type:      pipelineapi.ParamTypeString,
 						StringVal: "230",
 					},
+				},
+				pipelineapi.TaskRunResult{
+					Name: "shp-image-vulnerabilities",
+					Value: pipelineapi.ParamValue{
+						Type:      pipelineapi.ParamTypeString,
+						StringVal: vulns,
+					},
 				})
 
 			resources.UpdateBuildRunUsingTaskResults(ctx, br, tr.Status.Results, taskRunRequest)
 
 			Expect(br.Status.Output.Digest).To(Equal(imageDigest))
 			Expect(br.Status.Output.Size).To(Equal(int64(230)))
+			Expect(len(br.Status.Output.Vulnerabilities)).To(Equal(2))
```

Comment:
Nit: I recently learned there is a Ginkgo/Gomega idiomatic way to write that (didn't check the syntax again, but I think it is like this):
```suggestion
			Expect(br.Status.Output.Vulnerabilities).To(HaveLen(2))
```